app/models/like.rb

```diff
@@ -1,2 +1,4 @@
 class Like < ApplicationRecord
+  belongs_to :post
```

Comment:
- [ ] as part of the requirements the Like model should have a method that updates the likes counter for a post , kindly add that :pray: